juniper_official/Solutions/Traffic-Blackhole/chassis-alarms.rule

```diff
@@ -0,0 +1,119 @@
+/*
+	* REQ: COL-1109
+	* Rule to collect Chassis Alarms for Blackhole Detection RCA.
+*/
+healthbot {
+	topic hardware.fpc {
+		rule chassis-alarms {
```

Comment:
Based on the sensor paths, these seems to be cmerror's related data. Can we name the rule accordingly?

Reply:
Remaned the Rule to "cm-errors"